test/apps/react-oie/src/App.js

```diff
@@ -92,6 +92,7 @@ export default function App() {
     const newTransaction = flowMethod === 'idp' 
       ? await oktaAuth.idx.startTransaction() 
       : await oktaAuth.idx[flowMethod]();
+    console.log('Transaction:', newTransaction);
```

Comment:
remove this?

Reply:
It's in test app for debugging purpose, I added it on purpose, but in the future we can add some logging logic for idx flow, and hide it behind the `devMode` flag.